ElectronicObserver/Window/Tools/SortieRecordViewer/Sortie/Battle/Phase/PhaseSupport.cs

```diff
@@ -72,6 +78,15 @@ public PhaseSupport(SupportType apiSupportFlag, ApiSupportInfo apiSupportInfo, b
 
 			_ => -1,
 		};
+
+		if (apiSupportInfo.ApiSupportAiratack is ApiSupportAiratack airAttack)
```

Comment:
Resharper says to use not-null pattern here
Current code is fine to me, so I'll let you decide what to do

Reply:
Hmm, mine wasn't showing that, but no idea if I disabled it somehow or if I got an outdated version. Guess removing the extra variable makes sense.